CHANGELOG.md

```diff
@@ -6,20 +6,26 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
 **Features**
+* #4874 Allow joins in continuous aggregates
 * #5241 Allow RETURNING clause when inserting into compressed chunks
-* #5245 Mange life-cycle of connections via memory contexts
+* #5243 Enable real-time aggregation for continuous aggregates with joins
+* #5245 Manage life-cycle of connections via memory contexts
 * #5246 Make connection establishment interruptible
 * #5253 Make data node command execution interruptible
-* #5243 Enable real-time aggregation for continuous aggregates with joins
 * #5262 Extend enabling compression on a continuous aggregrate with 'compress_segmentby' and 'compress_orderby' parameters
 
 **Bugfixes**
 * #4926 Fix corruption when inserting into compressed chunks
-* #5218 Add role-level security to job error log
 * #5214 Fix use of prepared statement in async module
-* #5290 Compression can't be enabled on continuous aggregates when segmentby/orderby columns need quotation
+* #5218 Add role-level security to job error log
 * #5239 Fix next_start calculation for fixed schedules
+* #5290 Fix enabling compression on continuous aggregates with columns requiring quotation
+
++**Thanks**
+* @henriquegelio for reporting the issue on fixed schedules
 
 ## 2.9.3 (2023-02-03)
```

Comment:
@mahipv I can not add a comment for the 3 lines below -can you please also apply the following update?

```diff
-This release contains bug fixes since the 2.9.2 release.
-This release is high priority for upgrade. We strongly recommend that you
-upgrade as soon as possible.
+This release contains bug fixes since the 2.9.2 release and a fix for a security vulnerability (#5259).
+You can check the security advisory(https://github.com/timescale/timescaledb/security/advisories/GHSA-44jh-j22r-33wq)
+for more information on the vulnerability and the platforms that are affected.
+
+This release is high priority for upgrade. We strongly recommend that you upgrade as soon as possible.
```